book/0 - Introduction/1 - Introduction.html

```diff
@@ -22,9 +22,10 @@
 <ul>
 	<li><a href="http://cuberite.org/">Cuberite Homepage</a></li>
 	<li><a href="https://forum.cuberite.org/">Forum</a></li>
-	<li><a href="https://forum.cuberite.org/forum-2.html">Plugin Repository</a></li>
+	<li><a href="https://cuberite.org/plugins/">Plugin Repository</a></li>
```

Comment:
I'm not saying one or the other is better, simply that the link text is "Plugin Repository" so the href needs to be accurate.